src/app/article/article.scss

```diff
@@ -517,8 +520,8 @@ textarea::-webkit-input-placeholder {
     display: none;
   }
 
-  .container {
-    width: auto;
+  .main-cover .container {
+    width: 100% !important;
```

Comment:
important is not advised. Is there no way to get by what you are trying to fix here?